src/dos/dos.cpp

```diff
@@ -148,7 +150,47 @@ static uint16_t DOS_GetAmount(void) {
 #ifndef DOSBOX_CPU_H
 #include "cpu.h"
 #endif
-static inline void modify_cycles(Bits value) {
+
+// Taken from Dosbox-X
+int hdd_data_rate = 0; // 2.1MBytes/sec mid 1990s IDE PIO hard drive without
```

Comment:
No end of line comments, please, except in very special circumstances (typically for tabular data). Please move it to the preceding line.